docs/getting-started/migrating.md

```diff
@@ -0,0 +1,86 @@
+# Migrating from 0.X
+
+!!! tip
+    This page is up-to-date with version `1.0.0-beta.5` of bdk.
+
+So you're ready to migrate to BDK version 1.0, congratulations!
+This document contains some helpful tips that, with the help of some automation, should make the process as seamless as possible.
+
+The below steps are for migrating wallet details from the old [`bdk` v0.30][0] to the new [`bdk_wallet` v1.0][1].
+This procedure can be applied to wallets backed by a SQLite database.
+Of particular concern is the ability to restore the _last known address index_ for each keychain.
+This is important because without that metadata the new wallet may end up reusing receive addresses, which should be avoided for privacy reasons, although it should not cause loss of funds.
+
+!!! tip
+    NB: The migration process outlined below will not automatically restore the wallet's transaction data or local view of the blockchain.
+    Thanks to the public ledger however, we can restore all the pertinent information for this wallet using one of the blockchain client libraries supported by BDK.
+
+## Overview
+
+1. Load an old database
+1. Get last revealed addresses
+1. Create new wallet
+1. Restore revealed addresses
+1. Write to new database
+1. Sync
+
+<!-- overview -->
+```rust title="examples/rust/migrate-version/src/main.rs"
+--8<-- "examples/rust/migrate-version/src/main.rs:main"
+```
+
+## Walkthrough
+
+In a new rust project add these dependencies to Cargo.toml
+
+```toml
+[dependencies]
```

Comment:
since you have a Cargo.toml file you can pull these in from the file as a code snippet. you may need to pull the whole file (don't think we have an example of pulling a chunk from .toml yet), but it's still worth it imo to minimize potential for duplication errors when this page gets updated